test/integration/testConnectionWithOCSP.js

```diff
@@ -23,381 +22,261 @@ const testConnectionOptions = {
   sfRetryMaxLoginRetries: 2
 };
 
-const testRevokedConnectionOptions = {
-  accessUrl: 'https://revoked.badssl.com',
-  username: 'fakeuser',
-  password: 'fakepasword',
-  account: 'fakeaccount'
-};
+// const testRevokedConnectionOptions = {
```

Comment:
why commented, shouldn't we remove it if it's not used?

Reply:
I have uncommented the tests and skip them, but I think that fixing them should happen in a separate PR